frontend/src/components/Login.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+
+const SIGNUP_URL = 'https://project-auth-technigo.herokuapp.com/users' 
+const LOGIN_URL = 'https://project-auth-technigo.herokuapp.com/sessions'
+const SECRETS_URL ='https://project-auth-technigo.herokuapp.com/secrets'
+
+
+export const Login = () => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [secrets, setSecrets] = useState('')
+  const [status, setStatus] = useState('')
+
+  const validateEmptyMessage = message => message.replace(/\s/g, "").length === 0
+  const validateEmail = (email) => {
+    return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email)
+  }
+
+  const handleSignup = (event) => {
+    event.preventDefault()
+  
+    fetch(SIGNUP_URL, {
+      method: 'POST',
+      body: JSON.stringify({ email, password }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(res => res.json())
+      .then(json => {
+        setStatus(json.message)
+      })
+      .catch(err => console.log('error:', err))
+  }
+
+  const handleLogin = (event) => {
+    event.preventDefault()
+
+    fetch(LOGIN_URL, {
+      method: 'POST',
+      body: JSON.stringify({ email, password }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(res => res.json())
+      .then(json => {
+        handleSecrets(json.accessToken)
```

Comment:
The clue of this app was not to show something secret but create the logging system. Here you didn't save token anywhere. You created a situation where your users need to fill the form every time they want something from your server. 

At the beginning of the documentation is the information that you need to store your token in the local storage, and then you can use it without sending your form over and over again.